app/views/content/salaries-and-benefits.md

```diff
@@ -182,15 +161,3 @@ It is a 'defined benefit' pension and is:
 * flexible and allows you to take some of it as a tax-free lump sum
 
 You also get other insurance benefits too. Find out more about the [teachers' pension scheme](/what-pension-does-a-teacher-get).
-
-## Support for early career teachers
```

Comment:
@gemmadallmandfe @philspray I removed this section for multiple reasons:

a) the page is quite lengthy so already a lot for users to sift through
b) I feel like we lumped this in here when we created the new support for ECT page but that it doesn't really make sense on a page where we're talking about pay? and given the navigation work, we should be surfacing the support for ECT page in a much better way, so this feels a bit unnecessary now
c) looking in GA, VERY few users are actually clicking through to the ECT page link any way (0.15% of users in the last 6 months)

But again, happy to add back in if you think otherwise!

Reply:
If the data shows that very few users are clicking through, I think that's fine - however, we do need to make sure we are surfacing this page effectively elsewhere (which we should do as part of the nav work)